### ISSUE TITLE
[1337x] Exception (1337x): certificate validation failed: [Subject]  CN=*.safezone.mcafee.com, OU=Consumer, O=

### ISSUE THREAD

**Issue by SteffanCline** [Jackett/Jackett#12924]
### Have you checked our Troubleshooting page for your issue?

- [X] I have checked the Troubleshooting page

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Environment

```markdown
2022-02-05 20:11:16.3754 Info Starting Jackett v0.20.510 
2022-02-05 20:11:16.4281 Info Jackett Data will be stored in: /var/lib/jackett 
2022-02-05 20:11:16.9709 Info Environment version: 6.0.1 (/opt/Jackett/) 
2022-02-05 20:11:16.9709 Info OS version: Unix 5.11.0.49 (64bit OS) (64bit process) 
2022-02-05 20:11:16.9709 Info Jackett variant: CoreLinuxAmdx64 
2022-02-05 20:11:16.9709 Info File /etc/issue: Ubuntu 21.04 \n \l 
2022-02-05 20:11:16.9709 Info Running in Docker: No 
2022-02-05 20:11:16.9709 Info ThreadPool MaxThreads: 32767 workerThreads, 1000 completionPortThreads 
2022-02-05 20:11:16.9709 Info App config/log directory: /var/lib/jackett 
2022-02-05 20:11:16.9709 Info Using proxy: Disabled 
2022-02-05 20:11:16.9709 Info Using FlareSolverr: No 
2022-02-05 20:11:16.9754 Info Using HTTP Client: HttpWebClient2 
```

### Description

I'm not sure exactly what the cause is here. It's like this on many of the indexers on this fresh install of Ubuntu but these exact same trackers work fine on my MacOS install. I imported all the certs using all methods I could find on Google. Some of the docs talk about using regular mono versus mono-devel as the solution but they don't tell how. Simply running a test on several indexers triggers this exact issue.

What is the trick to fix this issue?

[log.txt](https://github.com/Jackett/Jackett/files/8013318/log.txt)

### Logged Error Messages

```text
Log file attached.
```

### Screenshots

<img width="518" alt="Screen Shot 2022-02-07 at 1 03 22 AM" src="https://user-images.githubusercontent.com/1323544/152748397-c03bac1b-3cd4-4e43-8513-f4d4050a8f7f.png">

**Comment by garfield69** [comment]
It appears to me like you have a mcafee firewall or other mcafee security app installed on your machine that is blocking your access to sites by replacing their CA with its own invalid mcafee certificate to force an interruption.

But your environment details you posted are confusing me as to which flavour of jackett you are running.
Please restart your jackett service and then access the jackett dashboard and view the log.
Select and copy the first 10 lines at the bottom of the log and paste them in the edited environment section of your original post. Thanks.
This better info may allow us to help you further if required.

**Comment by garfield69** [comment]
Oh, never mind, I see you provided the log, so I'll use that. Thanks.

**Comment by SteffanCline** [comment]
I am moving everything off MacOS onto Ubuntu 20. I copied the contents of the data folder into /var/lib/jackett, set permissions of course and set the config directory in the .service file. If it's relevant, in trying Prowalarr as well, it too fails on the same trackers so I suspect it's something related to this SSL issue. At this point, I think I'd rather stick with Jackett since I've been using it for years. This is running on my home network and there is no firewall turned on or anything in front of it other than NAT. I have no VPN or anything turned on. The MacOS install is using the same exact config, obviously different OS.

**Comment by garfield69** [comment]
The fact remains that Jackett is making a request to access a site, and something external to Jackett is blocking the access by providing a mcafee certificate instead of the one that comes from the site Jackett tried to access.

Usually the culprits are either: your ISP provider is blocking access, or your router security, or your ubuntu security.
And that is as much as I can tell you. 

@ilike2burnthing  Anything you would like to add to this, you know how I love these linux platforms ;-b

**Comment by garfield69** [comment]
As an experiment, consider shutting down Jackett, renaming that data folder and creating an empty replacement and restarting Jackett. 
Then add one of the previously unresponsive indexers afresh and see if it starts working. It will conform/eliminate the possibility the issue is with the imported macos data copy you performed.

**Comment by SteffanCline** [comment]
I checked my DSL router and they had their own DNS set. I changed it to use Google DNS to ensure that it's not as you suggested. I went back to the Mac and did a recheck on that same tracker as a test and it worked fine. I went back to Ubuntu and cleared the cache. I double checked that and it's statically configured to use Google DNS and is statically configured to an IP my home LAN. I renamed the Jackett directory, created an empty dir and restarted Jackett so it's starting out with completely fresh settings. I restarted Jackett and configured that same tracker as a test. It failed again with the same error. I rebooted the server and tested. ```ufw status verbose``` and it shows ```Status: inactive```. A manual test of the URL https://1337x.to/ works fine in a browser on my laptop.

I did a test from the Ubuntu server and get this:
```
curl -v https://1337x.to/
*   Trying 104.31.16.118:443...
* Connected to 1337x.to (104.31.16.118) port 443 (#0)
* ALPN, offering h2
* ALPN, offering http/1.1
* successfully set certificate verify locations:
*  CAfile: /etc/ssl/certs/ca-certificates.crt
*  CApath: /etc/ssl/certs
* TLSv1.3 (OUT), TLS handshake, Client hello (1):
* TLSv1.3 (IN), TLS handshake, Server hello (2):
* TLSv1.2 (IN), TLS handshake, Certificate (11):
* TLSv1.2 (IN), TLS handshake, Server key exchange (12):
* TLSv1.2 (IN), TLS handshake, Server finished (14):
* TLSv1.2 (OUT), TLS handshake, Client key exchange (16):
* TLSv1.2 (OUT), TLS change cipher, Change cipher spec (1):
* TLSv1.2 (OUT), TLS handshake, Finished (20):
* TLSv1.2 (IN), TLS handshake, Finished (20):
* SSL connection using TLSv1.2 / ECDHE-RSA-AES128-GCM-SHA256
* ALPN, server did not agree to a protocol
* Server certificate:
*  subject: C=US; ST=California; L=Santa Clara; O=McAfee, Inc.; OU=Consumer; CN=*.safezone.mcafee.com
*  start date: Aug 27 00:00:00 2021 GMT
*  expire date: Aug 27 23:59:59 2022 GMT
*  subjectAltName does not match 1337x.to
* SSL: no alternative certificate subject name matches target host name '1337x.to'
* Closing connection 0
* TLSv1.2 (OUT), TLS alert, close notify (256):
curl: (60) SSL: no alternative certificate subject name matches target host name '1337x.to'
More details here: https://curl.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.
```

I'm at a loss. I can't figure out where that SSL cert is coming from. I checked Chrome to see what it shows for SSL and I see this:

<img width="1285" alt="Screen Shot 2022-02-07 at 2 01 24 AM" src="https://user-images.githubusercontent.com/1323544/152757139-53a2899a-452d-47d8-94e2-eac2be3ab081.png">

**Comment by SteffanCline** [comment]
Ok, guys... I feel bad. I installed a UI on the server, opened Mozilla and queried the URL only to get the same thing. I queried part of the error and found this: https://github.com/FiloSottile/age/issues/370 which was in fact the case. I disabled that and reran the test and it worked! Sorry for the false alarm. So odd about this. I replaced my DSL modem a while back and the new one apparently has this "feature".
